### DISCUSSION TITLE
New application for Windows created

### DISCUSSION THREAD

**Discussion by georgeasiminei** [CoopNetwork/GMD-blockchain#16] [Announcements]
GMD node Windows desktop app was launched.  
Application will run both the node and an user interface on a PC. This user interface is the recommended way to use the GMD native wallet.

**Instructions:**
- Download desktop app installer, install it and run it (no Java dependency required)

**Links:**
- Download from here: https://node.thecoopnetwork.io:8443/GMD-Node-1.0.0%20Setup.exe
- Source code and details here: https://github.com/CoopNetwork/GMD-node-app

**Comment by LaurSmarandescu** [comment]
Thank you, guys! Keep up the good work! 👏👏🙏
#coopnetwork

**Comment by BebeB1321** [comment]
Salut ! cat dureaza KYC-ul pentru a primi hallmark pentru nod ?

**Comment by Stefandragoi10** [comment]
![2021-11-05_22-29_GMD-Node-1 0 0 Setup exe](https://user-images.githubusercontent.com/93795571/140574627-b753bfe8-cf17-444a-b113-1c97e01ca30c.jpg
![2021-11-05_22-29_GMD-Node-1 0 0 Setup exe](https://user-images.githubusercontent.com/93795571/140574654-8ba1b241-650a-403b-a72a-c08ed81aabe2.jpg)
)

**Reply by georgeasiminei** [reply]
@Stefandragoi10  Please press "Open Setup Log" and attach logs here.
This usually happens when an old installation of some other product using the squirrel installer did not finish properly.

First step is to try a restart. If it doesn't work, I will need the logs.
If you don't want to attach the logs here you can send them on Telegram to me: @zuper4
